doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/11_Preparing_for_V11.md

```diff
@@ -36,3 +36,94 @@
 - Replace deprecated JS functions
   - Use t() instead of ts()
   - Don't use pimcore.helpers.addCsrfTokenToUrl()
+- `JsonListing` class is deprecated. 
```

Comment:
For deprecations, I would just add a one liner to check upgrade(with link to upgrade notes) notes and adapt the code accordingly.